src/Widgets/PackageRow.vala

```diff
@@ -23,6 +23,9 @@ namespace AppCenter.Widgets {
         AbstractPackageRowGrid grid;
 
         public PackageRow.installed (AppCenterCore.Package package, Gtk.SizeGroup? info_size_group, Gtk.SizeGroup? action_size_group, bool show_uninstall = true) {
+#if POP_OS
+            selectable = false;
```

Comment:
Does the Pop!_OS theme have the right styling so you can see what row has keyboard focus? Otherwise, this will make keyboard navigation difficult.